src/pages/ProgramPage/ProgramContentListSection.tsx

```diff
@@ -23,10 +29,28 @@ const ProgramSectionBlock = styled.div`
   margin-bottom: 2.5rem;
 `
 const ProgramSectionTitle = styled.h3`
+  display: flex;
+  justify-content: space-between;
+  align-content: center;
   font-size: 20px;
   font-weight: bold;
 `
 
+const StyledPinnedIcon = styled.span<{ pin: boolean }>`
```

Comment:
It should be noted that when using style component to pass props, they must be strings, otherwise an error will appear in the browser's devtool. This is because the style component will receives all props and appears on the html structure.